Zend/tests/repeated_include/anonymous_class.phpt

```diff
@@ -0,0 +1,64 @@
+--TEST--
+Repeated include must not increase memory - anonymous class
+--SKIPIF--
+<?php
+// TODO test should pass even without opcache, but fixes are needed
+// related bug tracker https://bugs.php.net/bug.php?id=76982
+if (!extension_loaded('Zend OPcache') || !(opcache_get_status() ?: [])['opcache_enabled']) {
+    die('skip test currently requires opcache enabled');
+}
+?>
```

Comment:
I'd rather merge these test part of the OpCache fix or mark them as XFAIL, but then we're terrible at handling XFAIL tests

Reply:
I cannot provide the fix, why is waiting for the fix better than merging it now and asserting at least the zero-leak /w opcache?

XFAIL would be pretty annoying, but I would still prefer it than keeping this PR here forever.

Reply:
changed to XFAIL

Reply:
@Girgias updated, can you please approve/merge? Thank you.